CHANGES.md

```diff
@@ -1,4 +1,5 @@
 # 2.0.3 (UNRELEASED)
+- [FIXED] Allow pipes to be defined inside request event handlers.
```

Comment:
It doesn't really matter, but I think this could reasonably be a `[NEW]` or `[IMPROVED]` since it wasn't really a bug, more of a documented limitation.